skype/skype-ps/skype/Grant-CsTeamsAudioConferencingPolicy.md

```diff
@@ -0,0 +1,152 @@
+---
+external help file: MicrosoftTeams-help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/en-us/powershell/module/skype/grant-csteamsaudioconferencingpolicy
+schema: 2.0.0
+---
+
+# Grant-CsTeamsAudioConferencingPolicy
+
+## SYNOPSIS
+
+Assigns a Teams audio-conferencing policy at the per-user scope. Audio conferencing policies are used to manage audio conferencing toll- and toll-free numbers to be displayed in meeting invites created by users within your organization.
+
+## SYNTAX
+
+### GrantToTenant (Default)
+```
+Grant-CsTeamsAudioConferencingPolicy [-Global] [-PassThru] [[-PolicyName] <String>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### Identity
+```
+Grant-CsTeamsAudioConferencingPolicy [-PassThru] [[-PolicyName] <String>] [[-Identity] <String>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Granular control over which audio conferencing features your users can or cannot use is an important feature for many organizations. This cmdlet lets you assign a teams audio conferencing policy at the per-user scope. Audio conferencing policies determine the audio-conferencing toll- and toll-free numbers to be displayed in meeting invites created by users within your organization.
+
+## EXAMPLES
+
+### EXAMPLE 1
+```
+PS C:\> Grant-CsTeamsAudioCOnferencingPolicy -identity "Ken Myer" -PolicyName “Emea Users”
+```
+
+In this example, a user with identity "Ken Myer" is being assigned the “Emea Users” policy
+
+
+
+## PARAMETERS
+
+### -Global
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: GrantToTenant
+Aliases:
+
+Required: False
+Position: Named
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Identity
+Indicates the Identity of the user account the policy should be assigned to. User Identities can be specified using one of four formats: 1) the user's SIP address; 2) the user principal name (UPN); 3) the user's domain name and logon name, in the form domain\logon (for example, litwareinc\kenmyer); and, 4) the user's Active Directory display name (for example, Ken Myer). User Identities can also be referenced by using the user's Active Directory distinguished name.
+
+```yaml
+Type: System.String
+Parameter Sets: Identity
+Aliases:
+
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: True (ByPropertyName, ByValue)
+Accept wildcard characters: False
+```
+
+### -PassThru
+
```

Comment:
Please add a description

Reply:
"Enables you to pass a user object through the pipeline that represents the user account being assigned the online voice routing policy. "